<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiffColoursPage</class>
 <widget class="QWidget" name="DiffColoursPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure Diff colours&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line1">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QPushButton" name="headerButton">
       <property name="toolTip">
        <string>Select the background colour for header lines</string>
       </property>
       <property name="text">
        <string>Header Colour</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="headerSample">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Header Line</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="whitespaceButton">
       <property name="toolTip">
        <string>Select the background colour for bad whitespace</string>
       </property>
       <property name="text">
        <string>Whitespace Colour</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="whitespaceSample">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="textButton">
       <property name="toolTip">
        <string>Select the text foreground colour</string>
       </property>
       <property name="text">
        <string>Text Colour</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="textSample">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Normal Text</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="addedButton">
       <property name="toolTip">
        <string>Select the background colour for additions</string>
       </property>
       <property name="text">
        <string>Added Colour</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="addedSample">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Added Text</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="removedButton">
       <property name="toolTip">
        <string>Select the background colour for removed text</string>
       </property>
       <property name="text">
        <string>Removed Colour</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="removedSample">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Removed Text</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="replacedButton">
       <property name="toolTip">
        <string>Select the background colour for replaced text</string>
       </property>
       <property name="text">
        <string>Replaced Colour</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="replacedSample">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Replaced Text</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="contextButton">
       <property name="toolTip">
        <string>Select the background colour for context lines</string>
       </property>
       <property name="text">
        <string>Context Colour</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="contextSample">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Context Line</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>46</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
